testplan/report/testing/base.py

```diff
@@ -470,6 +470,7 @@ def __init__(self,
                  tags=None,
                  part=None,
                  fix_spec_path=None,
+                 env_status=None,
```

Comment:
Does it make sense for Report object to hold the status of env? I think all report related status should go to the Status class, maybe add the UNINIT state to indicate env is not started yet.

Reply:
I see where you are coming from and kind of agree that this isn't a perfect fit for the report object. However currently the design of the interactive mode is such that the interactive API reflects the report structure, so any information to be sent over the API should be included in the report.

The problem with adding another UNINIT status is that that doesn't allow the interactive UI to know when the environment is either STARTING or STOPPING - the env_status gives more information (STOPPED -> STARTING -> STARTED ->> STOPPING -> STOPPED cycle) versus a simple UNINIT -> READY switch. So I think for now it is best to keep the env_status.